### PR TITLE
drivers:interrupt: Improve PLIC with smp case

### PR DESCRIPTION
**Pull request by sycamoremoon**

### CODE REVIEW

File: src/drivers/interrupt/riscv_plic/riscv_plic.h [hunk 1]
```diff
@@ -8,6 +8,6 @@
 #ifndef IRQCTRL_RISCV_INTC_IMPL_H_
 #define IRQCTRL_RISCV_INTC_IMPL_H_
 
-#define __IRQCTRL_IRQS_TOTAL 70
+#define __IRQCTRL_IRQS_TOTAL 1023
```
Comment (anton-bondarev) [inline src/drivers/interrupt/riscv_plic/riscv_plic.h:11]: Maybe get it from an option?

**Review by anton-bondarev — APPROVED** [review]
LGTM

**Comment by anton-bondarev** [comment]
@sycamoremoon Thanks
I've approved it
Could you rebase it under embox/master

**Comment by sycamoremoon** [comment]
No problem

**Pull request merged**